templates/ts-apollo-postgres-backend/src/index.ts

```diff
@@ -26,7 +27,10 @@ const db = connectDB()
 const dbConfig = loadDBConfig()
 
 const { typeDefs, resolvers, contextCreator } = buildGraphbackAPI(modelDefs, {
-  dataProviderCreator: createKnexDbProvider(db)
+  dataProviderCreator: createKnexDbProvider(db),
+  plugins: [
```

Comment:
Let's remove it.

Reply:
Done, put it in to test multiple sets of resolvers.